changelog/fragments/run-local.yaml

```diff
@@ -0,0 +1,14 @@
+entries:
+  - description: >
+      Add 'run local' subcommand, which has the same functionality of the
+      deprecated 'run --local' mode.
```

Comment:
It is doing more than that. 
We need to put here the flags deprecated and the new subcommand added as well.